.travis.yml

```diff
@@ -84,7 +84,7 @@ before_script:
     - cp -R ./TestData/* ./TestCases/
 
     # Get the tutorial cases
-    - git clone -b develop https://github.com/su2code/su2code.github.io ./Tutorials
+    - git clone -b feature_external_def https://github.com/su2code/su2code.github.io ./Tutorials
```

Comment:
This should be fixed.

Reply:
Yep, we have the typical problem again where both need to be updated simultaneously. I'll push that change right before we merge this and update it.